practices/java-jdbc/test/java/com/uabc/dowa/jdbc/repository/StudentRepositoryTest.java

```diff
@@ -1,27 +1,48 @@
 package com.uabc.dowa.jdbc.repository;
 
+import com.uabc.dowa.jdbc.model.Student;
 import org.junit.Test;
 
-import static org.junit.Assert.fail;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
 
 public class StudentRepositoryTest {
     @Test
-    public void testAddStudent() {
-        fail("Implement me!");
+    public void testAddStudent() throws SQLException {
+        StudentRepository repo = new StudentRepository();
+        Student student = new Student();
+        student.setFirstName("Alejandro");
+        student.setLastName("Ramirez");
+        repo.persistStudent(student);
+        assertNotNull(repo.findStudentByFirstName(student.getFirstName()));
     }
 
     @Test
-    public void testFindAllStudents() {
-        fail("Implement me!");
+    public void testFindAllStudents() throws SQLException {
+        StudentRepository repo = new StudentRepository();
+        List <Student> allStuds = repo.findAllStudents();
+        //I am not really sure ih this is fine for the test
```

Comment:
You could add 3 and make sure you have those same 3, be careful of another test running before this, you would need to clear the db table before inserting